src/Security/InteractiveSecurityHelper.php

```diff
@@ -142,26 +143,36 @@ public function guessPasswordField(SymfonyStyle $io, string $userClass): string
         );
     }
 
+    /**
+     * @return AuthenticatorType[]
+     */
     public function getAuthenticatorClasses(array $firewallData): array
```

Comment:
This doesn't really return authenticator "classes" anymore, does it? Let's rename